testsuite/MDAnalysisTests/test_timestep_api.py

```diff
@@ -474,94 +474,94 @@ class TestTRJTimestep(_TestTimestep, _TRJTimestep):
-        assert_equal(ts.has_velocities, False)
-        assert_raises(NoDataError, getattr, ts, 'velocities')
-
-    def test_forces_remove(self):
-        # This test is different because TRR requires that the
-        # has flags get updated every frame
-        ts = self.Timestep(10, forces=True)
-        ts.frame += 1
-        ts.has_forces = True  # This line is extra for TRR
-        assert_equal(ts.has_forces, True)
-
-        ts.has_forces = False
-        assert_equal(ts.has_forces, False)
-        assert_raises(NoDataError, getattr, ts, 'forces')
-
-    def test_positions_expiry(self):
-        assert_equal(self.ts.has_positions, True)
-        assert_array_almost_equal(self.ts.positions, self.refpos)
-        self.ts.frame += 1
-        assert_equal(self.ts.has_positions, False)
-        assert_raises(NoDataError, getattr, self.ts, 'positions')
-
-    def test_velocities_expiry(self):
-        self.ts.velocities = self.refpos + 10
-        assert_equal(self.ts.has_velocities, True)
-        assert_array_almost_equal(self.ts.velocities, self.refpos + 10)
-        self.ts.frame += 1
-        assert_equal(self.ts.has_velocities, False)
-        assert_raises(NoDataError, getattr, self.ts, 'velocities')
-
-    def test_forces_expiry(self):
-        self.ts.forces = self.refpos + 100
-        assert_equal(self.ts.has_forces, True)
-        assert_array_almost_equal(self.ts.forces, self.refpos + 100)
-        self.ts.frame += 1
-        assert_equal(self.ts.has_forces, False)
-        assert_raises(NoDataError, getattr, self.ts, 'forces')
-
-    def test_positions_renewal(self):
-        assert_equal(self.ts.has_positions, True)
-        self.ts.frame += 1
-        assert_equal(self.ts.has_positions, False)
-        assert_raises(NoDataError, getattr, self.ts, 'positions')
-
-        self.ts.has_positions = True
-        assert_equal(self.ts.has_positions, True)
-        assert_equal(self.ts.positions, self.refpos)
-
-    def test_velocities_renewal(self):
-        self.ts.velocities = self.refpos + 10
-        assert_equal(self.ts.has_velocities, True)
-        assert_array_almost_equal(self.ts.velocities, self.refpos + 10)
-        self.ts.frame += 1
-        assert_equal(self.ts.has_velocities, False)
-        assert_raises(NoDataError, getattr, self.ts, 'velocities')
-
-        self.ts.velocities = self.refpos + 11
-        assert_equal(self.ts.has_velocities, True)
-        assert_array_almost_equal(self.ts.velocities, self.refpos + 11)
-
-    def test_forces_renewal(self):
-        self.ts.forces = self.refpos + 100
-        assert_equal(self.ts.has_forces, True)
-        self.ts.frame += 1
-        assert_equal(self.ts.has_forces, False)
-        assert_raises(NoDataError, getattr, self.ts, 'forces')
-
-        self.ts.forces = self.refpos + 101
-        assert_equal(self.ts.has_forces, True)
-        assert_array_almost_equal(self.ts.forces, self.refpos + 101)
-
+# class TestXTCTimestep(_TestTimestep, _XTCTimestep):
```

Comment:
According to this code that is actually what MDAnalysis is doing currently.

https://github.com/MDAnalysis/mdanalysis/blob/96e3ce9c289d784eae9ea6b0f7c70d05915cf170/package/MDAnalysis/coordinates/xdrfile/core.py#L245

in a convoluted way currently we are directly using for every save

```python
box = triclinic_vectors(triclinic_box(box_vectos))
```

But I'll definitely run a check how often that can be done until the errors sum.